stable/artifactory/templates/artifactory-access-config.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.access.tlsEnabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "artifactory.fullname" . }}-access-config
+  labels:
+    app: {{ template "artifactory.name" . }}
+    chart: {{ template "artifactory.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+type: Opaque
+stringData:
+  access.config.import.yml: |
+    security:
```

Comment:
I suggested getting the value from values.yaml, which will support more config in the future.

Reply:
Done